checkstyle/checkstyle.xml

```diff
@@ -69,7 +69,7 @@
         </module>
         <module name="RightCurly">
             <property name="id" value="RightCurlySame"/>
-            <property name="tokens" value="LITERAL_TRY, LITERAL_CATCH, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE, LITERAL_DO"/>
+            <property name="tokens" value="LITERAL_TRY, LITERAL_CATCH, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE"/>
```

Comment:
FYI, this fixes a bug in the version of checkstyle we are stuck on where it doesn't handle `do``while` loops properly.